app.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controller");
+
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.all("*", (req, res) => {
+  res.status(404).send({ msg: "endpoint not found" });
+});
+
+app.use((err, req, res, next) => {
```

Comment:
This is not required at this point, as app.all() will take care of any wrong paths.